src/phoenix/experimental/evals/models/rate_limiters.py

```diff
@@ -162,9 +163,9 @@ def __init__(
             rate_increase_factor=rate_increase_factor,
             cooldown_seconds=cooldown_seconds,
         )
-        self._rate_limit_handling = asyncio.Event()
-        self._rate_limit_handling.set()  # allow requests to start immediately
-        self._rate_limit_handling_lock = asyncio.Lock()
+        self._rate_limit_handling: Optional[asyncio.Event] = None
```

Comment:
These async primitves should be lazily created. We keep track of the event loop they are associated with so they can be regenerated if necessary.